src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        projects = {
+            d['id']: d
```

Comment:
OK — looked into this and it does look like it's done on some serializers. Personally, I don't like it (and can explain why if you care), but whether you want to change it or not is up to you.